packages/react-devtools-shared/src/devtools/views/Components/StackTraceView.js

```diff
@@ -22,7 +31,23 @@ type CallSiteViewProps = {
 };
 
 export function CallSiteView({callSite}: CallSiteViewProps): React.Node {
-  const symbolicatedCallSite: null | ReactCallSite = null; // TODO
+  const fetchFileWithCaching = useContext(FetchFileWithCachingContext);
+
+  const [virtualFunctionName, virtualURL, virtualLine, virtualColumn] =
+    callSite;
+
+  const symbolicatedCallSite: null | ReactFunctionLocation =
+    fetchFileWithCaching !== null
+      ? use(
```

Comment:
Do we give up on supporting older React versions for `react-devtools-inline`?

Reply:
What do you mean older React versions? It's all versions if the `react-devtools-inline` host isn't providing a way to fetch the files?

Older versions won't get any stacks anyway since it won't exist in the data.

Reply:
I didn't see those. Feels odd that we check in some places but not all e.g. https://github.com/facebook/react/blob/6aff312688be2e07a94de16bc141bd35a3fd11e3/packages/react-devtools-shared/src/devtools/views/ErrorBoundary/cache.js#L24